Client/Application/LeanplumIntegration.swift

```diff
@@ -117,6 +123,14 @@ class LeanplumIntegration {
         Leanplum.start(userAttributes: userAttributesDict)
 
         Leanplum.track(LeanplumEventName.openedApp.rawValue)
+
+        // Only drops Leanplum event when a user has installed Focus (from a fresh state or a re-install)
+        if profile?.prefs.boolForKey(PrefsKeys.HasFocusInstalled) == canInstallFocus() {
+            profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
+            if !canInstallFocus() {
+                Leanplum.track(LeanplumEventName.downloadedFocus.rawValue)
```

Comment:
This `.rawValue` can be removed if you call `LeanplumIntegration.track()` instead. Is there a reason not to do that?